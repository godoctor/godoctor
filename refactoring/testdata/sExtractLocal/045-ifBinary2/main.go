package main

import "fmt"

func main() {
	y := 2
	if y != 0 {
		x := 5
		if x < 10 { // <<<<< extractLocal,9,5,9,6,newVar,pass
			fmt.Println("divisible by 5:")
		}
	}
}
