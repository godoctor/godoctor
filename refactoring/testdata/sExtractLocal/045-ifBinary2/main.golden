package main

import "fmt"

func main() {
	y := 2
	if y != 0 {
		x := 5
		newVar := x
		if newVar < 10 { // <<<<< extractLocal,9,5,9,6,newVar,pass
			fmt.Println("divisible by 5:")
		}
	}
}
